etk-asm/src/asm.rs

```diff
@@ -15,23 +35,33 @@ impl From<TryFromIntError> for Error {
     }
 }
 
-#[derive(Debug, Default)]
```

Comment:
Why remove default?

Reply:
Adding back, not sure why I was thinking I needed to remove it.